### PR TITLE
fix configuration errors

### PR DESCRIPTION
**Pull request by whitewum**
fix disk storage size for 3 copies and memory size for reverse key

### CODE REVIEW

File: docs/manual-CN/3.build-develop-and-administration/3.configurations/0.system-requirement.md [hunk 1]
```diff
@@ -49,11 +49,12 @@
 * 内存：8 GB
 * 存储：100 GB, SSD
 
-## 资源估算
+## 资源估算（3副本标准配置）
 
-* 存储空间（全集群）：边数量 * 平均边属性的字节数 * 2.5
-* 内存（全集群）：边数量 * 4 字节 + RocksDB 实例数量 * (write_buffer_size * max_write_buffer_number + rocksdb_block_cache), 其中 `etc/nebula-storaged.conf` 文件中 `--data_path` 项中的每个目录对应一个 RocksDB 实例
+* 存储空间（全集群）：点和边数量 * 平均属性的字节数 * 6
+* 内存（全集群）：点边数量 * 5 字节 + RocksDB 实例数量 * (write_buffer_size * max_write_buffer_number + rocksdb_block_cache), 其中 `etc/nebula-storaged.conf` 文件中 `--data_path` 项中的每个目录对应一个 RocksDB 实例
```
Comment (laura-ding) [inline docs/manual-CN/3.build-develop-and-administration/3.configurations/0.system-requirement.md:55]: How did the 5 come from?

File: docs/manual-CN/3.build-develop-and-administration/3.configurations/0.system-requirement.md [hunk 2]
```diff
@@ -49,11 +49,12 @@
 * 内存：8 GB
 * 存储：100 GB, SSD
 
-## 资源估算
+## 资源估算（3副本标准配置）
 
-* 存储空间（全集群）：边数量 * 平均边属性的字节数 * 2.5
-* 内存（全集群）：边数量 * 4 字节 + RocksDB 实例数量 * (write_buffer_size * max_write_buffer_number + rocksdb_block_cache), 其中 `etc/nebula-storaged.conf` 文件中 `--data_path` 项中的每个目录对应一个 RocksDB 实例
+* 存储空间（全集群）：点和边数量 * 平均属性的字节数 * 6
```
Comment (laura-ding) [inline docs/manual-CN/3.build-develop-and-administration/3.configurations/0.system-requirement.md:54]: How did the 6 come from?

Comment (critical27) [reply docs/manual-CN/3.build-develop-and-administration/3.configurations/0.system-requirement.md:54]: 点 * 平均属性的字节数 * 3 + 边 * 平均属性的字节数 * 6 + wal大小？

File: docs/manual-CN/3.build-develop-and-administration/3.configurations/0.system-requirement.md (hunk 1 above)
Comment (whitewum) [reply docs/manual-CN/3.build-develop-and-administration/3.configurations/0.system-requirement.md:55]: bloomfilter's 2.5 * (inkey + outkey)

File: docs/manual-CN/3.build-develop-and-administration/3.configurations/0.system-requirement.md (hunk 2 above)
Comment (whitewum) [reply docs/manual-CN/3.build-develop-and-administration/3.configurations/0.system-requirement.md:54]: Generally, Edge count is much much larger than vertex count & wal. So I simplify the computation.

Comment (whitewum) [reply docs/manual-CN/3.build-develop-and-administration/3.configurations/0.system-requirement.md:54]: As in most cases, how many edges and vertices are only an estimation (e.g., about a 1 Billion). So I think it's not necessary to give a very accurate computation method.

**Pull request merged**